.github/workflows/mobilecoin-dev-cd.yaml

```
@@ -19,15 +26,20 @@ on:
     - release/*
     tags:
       - v[0-9]+*
+    paths-ignore:
+    - '**.md'
 
 # don't run more than one at a time for a branch/tag
-concurrency: mobilecoin-dev-cd-${{ github.ref }}
+concurrency:
+  group: mobilecoin-dev-cd-${{ github.head_ref || github.ref }}
```

`head_ref` doesn't seem right here -- IIUC this would result in two PRs to `master` clashing

```suggestion
  group: mobilecoin-dev-cd-${{ github.ref }}
```

---

NVM, I mixed up `head_ref` and `base_ref`